scripts/ParseNeededEnv.py

```diff
@@ -0,0 +1,26 @@
+import configparser
+from sys import argv
+import os
+
+# print(sys.argv)
+
+cfg_path = argv[1]
+if not os.path.isfile(cfg_path):
+    print("File was not found at {}".format(cfg_path))
+    exit(1)
+config = configparser.ConfigParser()
+
+try:
+    config.read(cfg_path)
+    base_env = config["DEFAULT"]["ENV_NAME"]
+except (KeyError, configparser.ParsingError) as error:
+    print("File at {} is not a valid config file.@{}".format(cfg_path, error))
+    exit(1)
+
+all_env = [base_env]
+for section in config.sections():
+    all_env.append(config[section]["ENV_NAME"])
+all_env = list(set(all_env))
+print(base_env)
```

Comment:
this printout is too much, breaks the setup script ?

Reply:
No, this is intended to deal with setups that use more than one env across their tasks. The first print provides the env that is supposed to be used for the local setup. Everything after the first is the full list of envs that will be needed and therefore set up. 
I could make it a bit shorter by only providing the second half and putting the starting env at the start of that list.

Reply:
for me, it did not work with KingMaker and both printouts

Reply:
Reworked in commit. Please let me know if issues persist on your side.